compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3878,13 +3875,11 @@ object Parsers {
         val ident = termIdent()
         var name = ident.name.asTermName
         val paramss =
-          if in.featureEnabled(Feature.clauseInterleaving) then
-            // If you are making interleaving stable manually, please refer to the PR introducing it instead, section "How to make non-experimental"
+          if Feature.clauseInterleavingEnabled(using in.languageImportContext) then
```

Comment:
like I said earlier, language imports and source versions should be kept separate 
```suggestion
          if sourceVersion.isAtLeast(`3.6`) || in.featureEnabled(Feature.clauseInterleaving) then
```